### PR TITLE
DOC: Added some information to README files

### PR DESCRIPTION
**Pull request by gtrabanco**
I added some information about installation and first steps in README files. The how to install in ARM macOS need testing because I do not have an ARM mac.

### CODE REVIEW

File: dotfiles_template/README.md [hunk 1]
```diff
@@ -4,10 +4,22 @@
...
+```bash
+export GITHUB_USER=""
+export DOTFILES_REPOSITORY="dotfiles"
+export DOTFILES_PATH="${HOME}/.dotfiles"
+export DOTLY_PATH="$DOTFILES_PATH/modules/dotly"
+#ssh-keygen # Uncoment to generate your ssh key
+#cat ~/.ssh/id_rsa.pub | pbcopy # Uncoment if you have generated a new key
+#      In Linux use 'xclip -selection clipboard'
+# Add to Github > Settings > SSH and GPG Keys > Add new SSH key
+xcode-select --install
```
Comment (rgomezcasas) [inline dotfiles_template/README.md:16]: This only works on macOS

File: dotfiles_template/README.md [hunk 2]
```diff
@@ -4,10 +4,22 @@
...
+#rm -rf "$DOTFILES_PATH" "~/.dotly" # Optional step
+git clone git@github.com:$GITHUB_USER/$DOTFILES_REPOSITORY.git "$DOTFILES_PATH"
+cd "$DOTFILES_PATH"
+git submodule update --init --recursive
+"$DOTLY_PATH/bin/dot" self install
+"$DOTLY_PATH/bin/dot" package import # Import your packages
+echo "Press a key to exit the terminal and later reopen it..."
+exit
+```
```
Comment (rgomezcasas) [inline dotfiles_template/README.md:25]: We could add all of this commands to a `script` and then execute it 😊

**Review by rgomezcasas — CHANGES_REQUESTED** [review]

File: dotfiles_template/README.md (hunk 1 above)
Comment (gtrabanco) [reply dotfiles_template/README.md:16]: yes, sorry forgot to addit.

File: dotfiles_template/README.md (hunk 2 above)
Comment (gtrabanco) [reply dotfiles_template/README.md:25]: Yes, its a work in progress in my repo but I'm out of time until half february. I was thinking about a remote "restorer" like the installer that do all the stuff.

Comment (gtrabanco) [reply dotfiles_template/README.md:25]: There is also a mac command to add if you approve PR #67

```bash
"$DOTLY_PATH/bin/dot" mac defaults import
```

**Review by gtrabanco — COMMENTED** [review]
See the new commits

**Pull request closed without merging**